core/core_modules/disable.js

```diff
@@ -1,24 +1,72 @@
-var disabled = false;
+// TODO Should send a different message if caused by the counter (should extract all messages into an array)
+
+// TODO Should be made configurable by sending /disable /counter <counterLimit> (any state is fine)
+// TODO Should flick disabled state using a timer by setting a timer /disable /timer <seconds> (any state is fine)
+// TODO Should be able to reset to default settings /disable /default (any state is fine, only defaults the configuration)
+
+// TODO Persistent config (counterLimit and disabled state) fairly acheivable by getting and setting properties in exports.platform.modules.disabledConfig
+// TODO Update local var setup after doing the above
+
+// TODO Should store disable states for each separate thread_id
+
+var prevTimeStamp;
+var isThreadDisabled = {},
+    possibleSpam = false;
+var counter = 0,
+    counterLimit = 3;
+var msgIndexDisable = 0,
+    msgIndexEnable = 0;
+var messages = [
+	    'Listen closely, take a deep breath. Calm your mind. You know what is best. What is best is you comply. Compliance will be rewarded. Are you ready to comply ',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs.](https://www.codacy.com/app/theprojectshop/Kassy/file/2533034220/issues/source?bid=3282987&fileBranchId=3282995#l20)